homeassistant/components/backup/config.py

```diff
@@ -37,7 +37,7 @@ class StoredBackupConfig(TypedDict):
     """Represent the stored backup config."""
 
     create_backup: StoredCreateBackupConfig
-    delete_after: StoredDeleteAfterConfig
+    backup_retention: StoredBackupRetentionConfig
```

Comment:
Maybe it should just be like this instead
```suggestion
    retention: StoredRetentionConfig
```

Reply:
I think it's clear what retention we're referring to without prefixing with backup. :+1: